package-manifest.toml

```diff
@@ -125,6 +125,7 @@ source.paths = [
   { from = "smf/nexus/{{rack-topology}}", to = "/var/svc/manifest/site/nexus" },
   { from = "out/console-assets", to = "/var/nexus/static" },
   { from = "schema/crdb", to = "/var/nexus/schema/crdb" },
+  { from = "tools/dtrace/nexus", to = "/var/nexus/dtrace" },
```

Comment:
FWIW, these really shouldn't go into `/var` -- they're not writeable data files, but program files.  They should go somewhere in `/opt`.

Reply:
I was blindly following the convention above. I'll change these ones, but should I also modify the others?

Reply:
At a glance those don't seem like writeable data files either, so I would be inclined to ship them under `/opt/oxide/omicron/share` or whatever, but I also have no idea what would break as part of fixing that.

Reply:
The files above are relied upon by other components, so I'll defer moving them another PR.

Reply:
https://github.com/oxidecomputer/omicron/issues/7254